app/src/ui/merge-branch/merge.tsx

```diff
@@ -92,15 +92,23 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
     }
   }
 
-  private renderMergeInfo() {
-
+  private renderMergeInfo(invalidBranch: boolean) {
     const commitCount = this.state.commitCount
-    const countPlural = commitCount === 1 ? 'commit' : 'commits'
-    const countText = commitCount === undefined
-      ? 'commits'
-      : <strong>{commitCount} {countPlural}</strong>
-
     const selectedBranch = this.state.selectedBranch
+    const currentBranch = this.props.currentBranch
+    
```

Comment:
Looks like you have some whitespace on this line, that's why it failed the build.